README.md

```diff
@@ -1 +1,63 @@
-# todo-list-v1
\ No newline at end of file
+
+# Project Name
+
+> This is basic version of todoList.
+
+
+## Built With
+
+- Major languages: JavaScript (ES6)
+- Frameworks: No framework only Vanilla JS
+- Technologies used: HTML, CSS, JavaScript, WebPack,
+
+## Live Demo
+
+[click here for demo]()
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- use git clone in order to get the copy this repo. You need to have atleast basic understanding of JavaScript.
+
+### Setup
+
+- You can simply open it on your vscode in order to start working on it
+
+### Install
+
+- please first install the node package files
+
```

Comment:
- [ ] Good job with your readme. But please add more instructions in order to help non-technical people to run your project locally :smile: 
```
git clone https://github.com/skyv26/todo-list-v1.git

cd todo-list-v1

npm install

npm run build 

npm start
```